bot/cogs/reddit.py

```diff
@@ -290,4 +290,8 @@ def channel(self) -> TextChannel:
 
 def setup(bot: Bot) -> None:
     """Load the Reddit cog."""
+    invalid_values = "", None
+    if any(value in (RedditConfig.secret, RedditConfig.client_id) for value in invalid_values):
```

Comment:
```suggestion
    if not RedditConfig.secret or not RedditConfig.client_id:
```
This do same and is more readable.